libs/libxx/Makefile

```diff
@@ -95,7 +97,7 @@ clean:
 	$(call DELFILE, $(BIN))
 	$(call CLEAN)
 
-distclean: clean
+distclean:: clean
```

Comment:
@xiaoxiang781216 users of other C++ libraries will need to be aware if this change, right?  They `distclean` would fail otherwise.

Reply:
I will provide the new patch for libcxx too. Like uClibc++, we don't need maintain the private copy anymore, the official release can be directly used.